workflow-to-js-transformer/src/test/java/com/github/cafdataprocessing/workflow/transform/WorkflowJavaScriptExecutionTest.java

```diff
@@ -732,6 +816,6 @@ private String getWorkflowJavaScriptFromXML(final String workflowXmlResourceIden
         final Path workflowXmlPath = Paths.get(testWorkflowXml.toURI());
 
         return WorkflowTransformer.transformXmlWorkflowToJavaScript(new String(
-                Files.readAllBytes(workflowXmlPath), StandardCharsets.UTF_8));
+                Files.readAllBytes(workflowXmlPath), StandardCharsets.UTF_8), PROJECT_ID);
```

Comment:
All tests are going to have this passed to the transform. What happens if they pass projectId as null? Is there checking to block that off and throw an exception before reaching the set params?

Reply:
I have added a test to ensure that this now throws a null pointer when a null projectId is passed to it.